backend/pipeline/mutator.go

```diff
@@ -59,7 +60,7 @@ func (p *Pipeline) mutateEvent(handler *corev2.Handler, event *corev2.Event) ([]
 		extension, err := p.store.GetExtension(tctx, handler.Mutator)
 		if err != nil {
 			if err == store.ErrNoExtension {
-				return nil, nil
+				return nil, fmt.Errorf("mutator '%s' does not exist", handler.Mutator)
```

Comment:
```suggestion
				return nil, fmt.Errorf("mutator %q does not exist", handler.Mutator)
```

Reply:
TIL `%q	a single-quoted character literal safely escaped with Go syntax.`

Reply:
The message looks funky with `%q` but will be more parseable with log management: `"error":"mutator \"nope\" does not exist"`.